libusb/os/windows_winusb.c

```diff
@@ -1208,7 +1253,7 @@ static int init_device(struct libusb_device *dev, struct libusb_device *parent_d
 			break;
 		}
 	} else {
-		r = init_root_hub(dev);
+		r = init_root_hub(dev, io_event);
 		if (r)
 			return r;
```

Comment:
in case if we early return here - `io_event` is not released

Reply:
fix all where io_event is not released